src/patternfly/components/Page/page-header-tools-group-item.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-page__header-tools-group-item{{#if page-header-tools-group-item--modifier}} {{page-header-tools-group-item--modifier}}{{/if}}"
```

Comment:
@mcoker were you saying `pf-c-page__header-tools-item` rather than `pf-c-page__header-tools-group-item`?
```suggestion
<div class="pf-c-page__header-tools-item{{#if page-header-tools-item--modifier}} {{page-header-tools-item--modifier}}{{/if}}"
```

Reply:
I think so, just because the class is getting really long, `group` in the name doesn't seem necessary, but I'm fine with it either way.

Reply:
I'm just thinking back to the old discussion, is a `group` wrapper `necessary` when there is nothing to group/items exists on its own..

Reply:
So adjacent groups and items would have the same spacing as groups currently, and items in groups would have no spacing (like how buttons, dropdowns, etc in groups now have no spacing)?

Reply:
Looks like that's how @christiemolloy set it up, so that groups with just 1 item are just an item, which is how I would expect it. Though I think we should apply the same `margin-left` that is on groups to items that are adjacent to a group, then disable that margin on the item with `.pf-m-avatar`, since we'll need to leave the `margin-left` on `.pf-c-avatar` so as not to make this a breaking change. Then in a breaking change release we can remove the margin on `.pf-c-avatar` and apply that to the item with `.pf-m-avatar`. What do you think?

Reply:
Ah yes I meant to do `pf-c-page__header-tools-item` , which works better because the item doesn't necessarily need to be part of the group.

Reply:
@mcoker I think that icon button groups, because there is no border on the button and padding makes them appear to be nicely spaced, are an exception to what might appear in the page header tools group. So, target first child and apply large spacer value, then inside groups apply spacer value (defined by design) to children. The exception would be group of things that should not have margins, like `-m-icons`. 

![Screen Shot 2019-07-11 at 11 50 35 AM](https://user-images.githubusercontent.com/5385435/61066850-5078d080-a3d4-11e9-9951-bfa6608e1628.png)
![Screen Shot 2019-07-11 at 12 06 14 PM](https://user-images.githubusercontent.com/5385435/61066851-5078d080-a3d4-11e9-8d4f-bce0bf95dac3.png)

What do you think?

Reply:
I suppose it depends on what all goes in the masthead. If we're adding buttons, I wonder if that would be another case where the action list layout would be used.